Sources/LiveViewNative/Coordinators/LiveSessionCoordinator.swift

```diff
@@ -107,20 +107,21 @@ public class LiveSessionCoordinator<R: RootRegistry>: ObservableObject {
         $navigationPath.scan(([LiveNavigationEntry<R>](), [LiveNavigationEntry<R>]()), { ($0.1, $1) }).sink { [weak self] prev, next in
             guard let self else { return }
             Task {
+                let last_channel = prev.last?.coordinator.liveChannel!
```

Comment:
This should be `lastChannel` in camelCase

Reply:
my poor rust addled brain.